### ISSUE TITLE
Keyboard for Android

### ISSUE THREAD

**Issue by Sorroko** [Sorroko/cclite#58]
Since the android soft keyboard lacks features, some extra keys would be useful such as the ctrl key and arrow keys.

A good way to do this would be as in the JuiceSSH app

![image](https://cloud.githubusercontent.com/assets/1191498/7004754/ff88a908-dc67-11e4-8549-09ebd0b2877c.png)

However the issue is that there is currently no way to determine the height of the keyboard and therefore no way to position the buttons just above the keyboard. It may be easier to fork the love-android-sdl2 project and add these features as well as removing unnecessary permissions.

**Comment by RamiLego4Game** [comment]
As i was thinking, androids offical keyboard is not enough for a cc lite, that's why porting to android will take me alot of time,  i'll make a custome android keyboard that supports multitouch.

**Comment by Sorroko** [comment]
I agree, the keyboard doesn't allow you to use all of cc properly, however I think it's simpler to do something as shown in the screenshot as making a custom keyboard is a lot more work and requires a separate installation

**Comment by RamiLego4Game** [comment]
Now when i finish my hw, then i'll start working on, i may have an alpha version of the keyboard by working for 2 days

**Comment by Sorroko** [comment]
Yeah, it should be simple enough to add some buttons for now
